source/auth/auth.rst

```diff
@@ -1193,6 +1194,9 @@ Q: Why does SCRAM sometimes SASLprep and sometimes not?
 Version History
 ===============
 
+Version 1.11.0 Changes
+    * Support speculative authentication via handshake starting in version 4.4 of the server.
+
```

Comment:
It looks like the main content was moved to `mongodb-handshake/handshake.rst`. This file doesn't have anything more than a changelog entry and version bump in its header. Perhaps these should be removed?

Reply:
I have reverted the authentication spec.